src/components/footer/pager.component.ts

```diff
@@ -1,26 +1,35 @@
 import {
   Component, Input, Output, EventEmitter, ChangeDetectionStrategy
 } from '@angular/core';
+import { Keys } from '../../utils';
 
 @Component({
   selector: 'datatable-pager',
   template: `
     <ul class="pager">
-      <li [class.disabled]="!canPrevious()">
-        <a
+      <li
+        [class.disabled]="!canPrevious()"
+        (keypress)="onKeyPress($event, firstPageButton)"
+        [tabindex]="(canPrevious() ? '0' : '-1')">
+        <a #firstPageButton
           role="button"
           aria-label="go to first page"
           href="javascript:void(0)"
-          (click)="selectPage(1)">
+          (click)="selectPage(1)"
+          tabindex="-1">
           <i class="{{pagerPreviousIcon}}"></i>
         </a>
```

Comment:
Hi great work here improving the accessibility of ngx-databale. A suggestion here would be to use buttons instead of a tags. 

I am currently working on improving the accessibility of Ceph dashboard which uses ngx-datatable and we found that the a tags are not navigable with keyboards and a button would work better. You can take a look at discussion in this [pull request](https://github.com/ceph/ceph/pull/46958)